QPong-Unity-Version/Assets/Scripts/Definitions.cs

```diff
@@ -2,6 +2,7 @@
 
 // Game enums
 public enum Scenes { MainMenu = 0, Game, Ranking, Credits};
+public enum ArcadeButtonGates { None, iz, zi, ih, hi, ix, xi, cz };
```

Comment:
At this time game buttons there are only H & X gates, right?

Reply:
yes that is true but the buttons will be the same as Hello Quantum so i wanted to include them in the game in case we have a use for them